### PR TITLE
Update nxos_aaa_server_host.py

### PR DESCRIPTION
**Pull request by QCU87Z**
Fixed typos in the Examples.
Missing quotes and curly braces

+label: docsite_pr

##### SUMMARY
Fixed some issues in the example section of the documentation.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
nxos_aaa_server_host

##### ADDITIONAL INFORMATION
The examples would not work in there current state.

### CODE REVIEW

File: lib/ansible/modules/network/nxos/nxos_aaa_server_host.py [hunk 1]
```diff
@@ -106,9 +106,9 @@
         tacacs_port: 89
         host_timeout: 10
         address: 5.6.7.8
-        host:  inventory_hostname }}
-        username:  un }}
-        password:  pwd }}
+        host: "{{ inventory_hostname }}"
+        username: "{{ un }}"
+        password: "{{ pwd }}"
```
Comment (trishnaguha) [inline lib/ansible/modules/network/nxos/nxos_aaa_server_host.py]: Thanks for the PR! The top level spec actually needs to removed from example as we have deprecated them.

**Review by trishnaguha — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/nxos/nxos_aaa_server_host.py (hunk 1 above)
Comment (samccann) [reply lib/ansible/modules/network/nxos/nxos_aaa_server_host.py]: @trishnaguha - can you add more detail to your request? This came in as a docsite_pr so the owner may not follow what you are suggesting (and I'm not sure what you want changed either :-)  Thanks!

Comment (samccann) [reply lib/ansible/modules/network/nxos/nxos_aaa_server_host.py]: okay read the page a bit more - are you saying host, username, and password should be removed from the examples entirely?

Comment (trishnaguha) [reply lib/ansible/modules/network/nxos/nxos_aaa_server_host.py]: @samccann Yes that's correct. It should be removed from the task example entirely. As we have deprecated top level spec we would not like to encourage user to use it.

**Comment by samccann** [comment]
Thanks @QCU87Z for your help on the Ansible documentation!

**Pull request merged**